### DISCUSSION TITLE
New release possible for failsafe adapter?

### DISCUSSION THREAD

**Discussion by gstokkink-bex** [jnunemaker/flipper#635] [General]
It would be very useful to us, thanks in advance!

**Comment by bkeepers** [comment]
Hey @gstokkink, thanks for asking. i just did a bunch of work today to merge some other PRs. We'll get a new release out soon.

**Reply by gstokkink-bex** [reply]
@bkeepers Thanks for the update! Loving this gem, keep up the great work.

**Comment by bkeepers** [comment]
0.25.0 is out!

**Reply by gstokkink-bex** [reply]
@bkeepers awesome, thanks!
